code/get_analysis.py

```diff
@@ -73,8 +75,25 @@ def viewRemaining(chat_id, bot):
         bottom += weight_count
 
     ax.set_title("Category-wise budget consumed")
+    plt.xlabel("Categories")
+    plt.ylabel("Percentage")
     ax.legend(loc="upper right")
 
+    random_time = time.time()
+    img_name = str(random_time)+".png"
+    plt.savefig(img_name)
+    bot.send_photo(chat_id, photo=open(img_name, 'rb'), reply_markup=types.ReplyKeyboardRemove())
+
+def viewHistory(chat_id, bot):
+    if not helper.getUserHistory(chat_id):
+        bot.send_message(chat_id, "No history available", reply_markup=types.ReplyKeyboardRemove())
+    
```

Comment:
control continues. Have to add return statement

Reply:
mb added now.